ssn/ssn_separated/ssn.ttl

```diff
@@ -567,38 +487,35 @@ But the two express the relationship between constraints and multiple measuremen
+                          rdfs:comment """Collects together measurement properties (Accuracy, MeasurementRange, Precision, etc) and the environmental conditions in which those properties hold, representing a specification of a Sensor's or Actuator's capability in those conditions.
 
-The conditions specified here are those that affect the measurement properties, while those in OperatingRange represent the sensor's standard operating conditions, including conditions that don't affect the observations.""" ;
+The conditions specified here are those that affect the measurement properties, while those in OperatingRange represent the sensor's standard operating conditions, including conditions that don't affect the Observations.""" ;
                           rdfs:isDefinedBy "http://www.w3.org/ns/ssn" ;
-                          rdfs:label "Measurement Capability" ;
-                          rdfs:seeAlso "http://www.w3.org/2005/Incubator/ssn/wiki/SSN_Sensor#MeasuringCapability" .
+                          rdfs:label "Measurement Capability" .
 
 
 ###  http://www.w3.org/ns/ssn/MeasurementProperty
 ssn:MeasurementProperty rdf:type owl:Class ;
                         rdfs:subClassOf ssn:Property ;
-                        rdfs:comment "An identifiable and observable characteristic of a sensor's observations or ability to make observations." ;
+                        rdfs:comment "An identifiable and observable characteristic of a Sensor's observations or ability to make Observations." ;
                         rdfs:isDefinedBy "http://www.w3.org/ns/ssn" ;
-                        rdfs:label "Measurement Property" ;
-                        rdfs:seeAlso "http://www.w3.org/2005/Incubator/ssn/wiki/SSN_Sensor#MeasuringCapability" .
+                        rdfs:label "Measurement Property" .
 
 
 ###  http://www.w3.org/ns/ssn/MeasurementRange
 ssn:MeasurementRange rdf:type owl:Class ;
                      rdfs:subClassOf ssn:MeasurementProperty ;
                      dc:source """skos:exactMatch 'measuring interval/measurement range' [VIM 4.7]
                                   http://www.bipm.org/utils/common/documents/jcgm/JCGM_200_2008.pdf""" ;
-                     rdfs:comment "The set of values that the sensor can return as the result of an observation under the defined conditions with the defined measurement properties.  If no conditions are specified or the conditions do not specify a range for the observed qualities, the measurement range is to be taken as the condition for the observed qualities." ;
+                     rdfs:comment "The set of values that the sensor can return as the Result of an Observation under the defined conditions with the defined MeasurementProperties. If no Conditions are specified or the Conditions do not specify a range for the observed qualities, the MeasurementRange is to be taken as the Condition for the observed qualities." ;
```

Comment:
observed ObservableProperty ?

try to avoid the use of "quality", which was strongly related to the DUL class